src/compute-pr-actions.ts

```diff
@@ -394,6 +379,34 @@ export function process(prInfo: PrInfo | BotEnsureRemovedFromProject | BotNoPack
     return context;
 }
 
+function makeStaleness(now: string, author: string, otherOwners: string[]) { // curried for convenience
+    return (kind: StalenessKind, since: Date,
+            freshDays: number, attnDays: number, nearDays: number,
+            doneColumn: ColumnName | "CLOSE") => {
+        const days = daysSince(since, now);
+        const state = days <= freshDays ? "fresh" : days <= attnDays ? "attention" : days <= nearDays ? "nearly" : "done";

```

Comment:
```suggestion
        const state: Staleness["state"] = days <= freshDays ? "fresh" : days <= attnDays ? "attention" : days <= nearDays ? "nearly" : "done";
```
makes the `as Staleness` type assertion unnecessary? Too bad it's not inferred.

Reply:
Not needed (see below).